### DISCUSSION TITLE
Any website or tools that you can recommend for practicing miryoku layout?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [manna-harbour/miryoku#263] [Q&A] [answered]
curretly i am using keybr.com, but it dosent have numbers mixed with alphabets. i wanted to know whick website/tool did you use to be fluent miryoku layout.

**Comment by extua** [comment, marked as answer]
The typing tests on [monkeytype](https://monkeytype.com/) have an option for numbers and punctuation.

**Reply by KAGEYAM4** [reply]
i will do that. thanks.

**Reply by KAGEYAM4** [reply]
@extua if you don't mind can you give your suggestion on https://github.com/manna-harbour/miryoku/discussions/261

**Reply by manna-harbour** [reply]
To learn a new alpha layout I recommend using keybr initially, then switching to monketype. 

For the Sym layer I recommend drilling with monkeytype.
